src/routes/version.js

```diff
@@ -0,0 +1,22 @@
+const { auth } = require('../middlewares')
+const { StatusCodes } = require('http-status-codes')
+const { getLatestMobileVersion } = require('../controllers/version')
+const appGetter = require('../app')
+
+module.exports = () => {
+  const app = appGetter()
+
+  app.route({
+    method: 'GET',
+    url: '/mobile-app-latest-version',
```

Comment:
Maybe simple `/version` could do the trick?
I doubt we'd need any other version in this project.